packages/flutter_tools/lib/src/commands/doctor.dart

```diff
@@ -14,6 +14,11 @@ class DoctorCommand extends FlutterCommand {
       negatable: false,
       help: 'Run the Android SDK manager tool to accept the SDK\'s licenses.',
     );
+    argParser.addFlag('fuchsia',
+      defaultsTo: false,
+      negatable: false,
+      hide: true
```

Comment:
This should be `!verbose` or `!verboseHelp` or something (check the other commands for the pattern), and it should be documented with a `help` argument. We do want it to show up if you do `flutter --help --verbose`